.travis.yml

```diff
@@ -11,3 +11,7 @@ branches:
 cache:
   directories:
     - node_modules
+env:
+  global:
+    - SAUCE_USERNAME=cerner
```

Comment:
I'm not sure the username needs to (especially since it's so easy to guess). The key on the next line is encrypted with this repo's private key so that will keep anyone from using it outside this repo.